src/PhpSpec/Runner/Maintainer/CollaboratorsMaintainer.php

```diff
@@ -120,18 +120,8 @@ public function getPriority()
      */
     private function generateCollaborators(CollaboratorManager $collaborators, \ReflectionFunctionAbstract $function, \ReflectionClass $classRefl)
     {
-        if ($comment = $function->getDocComment()) {
-            $comment = str_replace("\r\n", "\n", $comment);
-            foreach (explode("\n", trim($comment)) as $line) {
-                if (preg_match(self::$docex, $line, $match)) {
-                    $collaborator = $this->getOrCreateCollaborator($collaborators, $match[2]);
-                    $collaborator->beADoubleOf($match[1]);
-                }
-            }
-        }
-
+        $foundClasses = array();
```

Comment:
this variable name is misleading. It does not contain classes, but collaborator names